configuration/addresshierarchy/liberia_address_hierarchy_entries.csv

```diff
@@ -11531,9 +11537,13 @@ Liberia|Montserrado|Greater Monrovia|Barekling
 Liberia|Montserrado|Greater Monrovia|Bassa Town Caldwell
 Liberia|Montserrado|Greater Monrovia|Battery Factory
 Liberia|Montserrado|Greater Monrovia|Bertha
-Liberia|Montserrado|Greater Monrovia|Bong Mine Bridge
```

Comment:
Just a small side note... addresses are not stored by key in the database, so just as an fyi, changing this won't update the existing addresses in the system, it will just affect new ones.  If it's important to change old ones, we'd need a liquibase change set.

Reply:
You are welcome!  I assume you have merge privileges on this repo, but if you need me to merge, let me know!

Reply:
Once it looks good, will merge. It's okay to merge?

Reply:
Thanks @mogoodrich